samples/tutorials/mnist/01_Lightweight_Python_Components.ipynb

```diff
@@ -104,12 +104,32 @@
    "outputs": [],
    "source": [
     "# Optional Parameters, but required for running outside Kubeflow cluster\n",
+    "\n",
+    "# The host for 'managed AI Platform Pipeline' ends with 'pipelines.googleusercontent.com'\n",
```

Comment:
Can we call it 'AI Platform Pipelines' (without managed)?

Reply:
Sure, changed all the occurrence.

Reply:
Hi @numerology , would you take a look whether this PR is alright?